lib/tarantool_server.py

```diff
@@ -38,12 +40,15 @@
 def save_join(green_obj, timeout=None):
     """
     Gevent join wrapper for
-    test-run stop-on-crash feature
+    test-run stop-on-crash/stop-on-timeout feature
 
-    :return True in case of crash and False otherwise
+    :return True in case of crash or test timeout and False otherwise
     """
     try:
-        green_obj.join(timeout=timeout)
+        green_obj.get(timeout=timeout)
+    except Timeout:
+        color_stdout("Test timeout of %d secs reached\t" % timeout, schema='error')
```

Comment:
That looks as a bit wrong abstraction level for this message, however okay as the temporary solution. Just noted for myself, we'll possibly need to revisit it.